### DISCUSSION TITLE
Getting style of SelectionList and ListView to match

### DISCUSSION THREAD

**Discussion by needsleep55** [Textualize/textual#3138] [Q&A]
Hi, I'm wondering what the easiest way to get a SelectionList and ListView to be as identical as possible given the same list of items. Is this possible?

Thanks.

**Comment by davep** [comment]
Do you think you could provide a small self-contained example of what you're trying to do, and give some sort of indication of the result you're looking for?

**Reply by needsleep55** [reply]
```python
from textual.app import App, ComposeResult
from textual.widgets import ListItem, ListView, SelectionList, Button, ContentSwitcher, Label
from textual.containers import Container


class ListStyleApp(App):

    def compose(self) -> ComposeResult:
        test_list = [
            ("Test 1", 1),
            ("Test 2", 2),
            ("Test 3", 3),
            ("Test 4", 4),
        ]

        list_view_content = [ListItem(Label(t[0])) for t in test_list]

        sel_list = SelectionList(*test_list, id="sel_list")
        with Container():
            with ContentSwitcher(id="switcher", initial=sel_list.id):
                yield sel_list
                yield ListView(*list_view_content, id="list_view")
        yield Button("Switch", id="run_btn")
        self.set_focus(sel_list)

    def on_button_pressed(self, event: Button.Pressed) -> None:
        switcher = self.query_one("#switcher")
        sel_list = self.query_one("#sel_list")
        if switcher.current == sel_list.id:
            switcher.current = self.query_one("#list_view").id
        else:
            switcher.current = sel_list.id


if __name__ == "__main__":
    app = ListStyleApp()
    app.run()
```

Here's something to kind of show what I'm going for. I have a small app that's supposed to allow a user to select a set of diagnostic tests to run. After hitting the run button, it will switch to a ListView (because I want to add some more widgets to the ListItems) to show the results of each test.

I guess what I'm asking is how, if possible, would I get the SelectionList and ListView to look as similar as possible. I'm not knowledgable with CSS so I'm not quite sure how to go about this.

Please let me know if I should be more clear.

**Reply by davep** [reply]
It might be helpful if you could highlight the differences you're seeing that you want to handle.
